### PR TITLE
Zap thunderbird configuration and cache files.

### PR DESCRIPTION
**Pull request by renard**

### CODE REVIEW

File: Casks/thunderbird.rb [hunk 1]
```diff
@@ -9,4 +9,12 @@
   tags :vendor => 'Mozilla'
 
   app 'Thunderbird.app'
+
+  zap :delete => [
+        '~/Library/Thunderbird',
+        '~/Library/Caches/Thunderbird',
+        '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/org.mozilla.thunderbird.sfl',
+        '~/Library/Saved Application State/org.mozilla.*',
```
Comment (vitorgalvao) [inline Casks/thunderbird.rb]: `:delete` doesn’t support expansion. You’ll need to be more specific.

You’re being too broad, anyway, this looks like it’ll delete saved states for other Mozilla apps, such as Firefox.

Comment (renard) [inline Casks/thunderbird.rb]: Ok so line 17 `'~/Library/Saved Application State/org.mozilla.*',` could be removed from patch.
Do you want me to fix that?

Comment (vitorgalvao) [inline Casks/thunderbird.rb]: Yes, one of them should go (why are you repeating it?) and the other should point to a specific file.

Comment (renard) [inline Casks/thunderbird.rb]: I am not sure to understand your meaning: "one of them should go (why are you repeating it?) and the other should point to a specific file"

There are actually several things to remove:
- `~/Library/Caches/Thunderbird/`
- `~/Library/Thunderbird/`
- `~/Library/Saved Application State/org.mozilla.thunderbird.savedState/`
- `~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/org.mozilla.thunderbird.sfl`


This is found using:
`find ~/Library | grep -i thunderbird`

Comment (vitorgalvao) [inline Casks/thunderbird.rb]: That’s exactly what it should be. That last comment.

Comment (renard) [inline Casks/thunderbird.rb]: ok should be good now.

**Pull request merged**